src/components/Header/Header.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  FC,
+  Dispatch,
+  SetStateAction,
+  useState,
+  useRef,
+  useEffect,
+} from 'react';
+import cn from 'classnames';
+
+import { Todo } from '../../types/Todo';
+import { ErrorMessages } from '../../types/ErrorMessages';
+import { postTodo, USER_ID } from '../../api/todos';
+
+interface Props {
+  setTempTodo: Dispatch<SetStateAction<Todo | null>>;
+  setTodos: Dispatch<SetStateAction<Todo[]>>;
```

Comment:
It will be more readability. Use this syntax for all the cases in the future
```suggestion
  setTempTodo: (tempTodo: Todo | null) => void;
  setTodos: (todos: Todo[]) => void;
```